datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/jmx/{check_name}/README.md

```diff
@@ -50,8 +50,8 @@ Need help? Contact [Datadog support][4].
 
 [1]: **LINK_TO_INTEGERATION_SITE**
 [2]: https://github.com/DataDog/integrations-{repo_choice}/blob/master/{check_name}/datadog_checks/{check_name}/data/conf.yaml.example
-[3]: https://docs.datadoghq.com/integrations/java/
-[4]: https://docs.datadoghq.com/help/
-[5]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
-[6]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
+[3]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
+[4]: https://docs.datadoghq.com/integrations/java/
```

Comment:
Looks like on line 48, the link needs to be updated since [4] doesn't go to the help page anymore. It may be easier just to keep the ordering as is to reduce changing things too much?